Resources/config/services.xml

```diff
@@ -54,7 +61,7 @@
             <argument type="service" id="security.helper" />
         </service>
 
-        <!-- Formators -->
+        <!-- Formations -->
```

Comment:
Asi je to hloupost, ale nedáme spíš `Formatters` nebo něco takového? Ale je to fuk asi:)

Reply:
Prejmenovano